.prettierrc.cjs

```diff
@@ -0,0 +1,5 @@
+const prettierConfig = require('./packages/eslint-config/prettier');
```

Comment:
I renamed this file in #719!

```suggestion
const prettierConfig = require('./packages/eslint-config/prettier.config');
```

Reply:
Nice, will do!

Reply:
Updated this in https://github.com/readmeio/standards/pull/717/commits/94441d20ac17ec14a74d4240e9126bfaf92e8aa2 and https://github.com/readmeio/standards/pull/717/commits/5be08b4f7f0a781e66b9e256bb7a92cb9c085cca